src/main/java/uk/gov/hmcts/probate/controller/BusinessValidationController.java

```diff
@@ -137,16 +138,19 @@ public ResponseEntity<CallbackResponse> validateFurtherEvidence(@RequestBody Cal
 
     @PostMapping(path = "/sols-create-validate-default-iht-estate", consumes = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<CallbackResponse> validateSolsCreateDefaultIhtEstate(
-            @Validated({ApplicationCreatedGroup.class}) @RequestBody
-                    CallbackRequest callbackRequest) {
+            @RequestBody CallbackRequest callbackRequest) {
 
-        final List<ValidationRule> solPcValidation = Arrays.asList(solicitorPostcodeValidationRule);
+        if (callbackRequest.getCaseDetails().getData().getApplicationType().toString()
```

Comment:
I dont think this is correct
You probably only need to add the solicitor condition to the rule you already made, so that it only errors for appType=Solicitor